com.dynamo.cr/com.dynamo.cr.bob/src/com/dynamo/bob/Bob.java

```diff
@@ -150,10 +151,10 @@ public static void init() {
         } catch (Exception e) {
             throw new RuntimeException(e);
         }
-        TimeProfiler.stop();
+        scope.stop();
     }
 
-    public static void initLua() {
+    public static synchronized void initLua() {
```

Comment:
Synchronize this so that multiple threads don't try to unpack Lua at the same time